app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -22,6 +24,33 @@ class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer
     end
   end
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
+          :label      => _('Container Name'),
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
+        },
+        {
+          :component    => 'select',
+          :name         => 'provider_region',
+          :id           => 'provider_region',
```

Comment:
@GilbertCherrie the create method below expects this to be `options[:create_bucket_configuration][:location_constraint]` does that need to change?
Also that says it is a location_constraint so should this be optional?

Reply:
@agrare This is all handled in the ui and api code to convert it to `options[:create_bucket_configuration][:location_constraint]`. Also, I made it required because the old ui form had it required.

Reply:
> This is all handled in the ui and api code to convert it to options[:create_bucket_configuration][:location_constraint].

That feels very amazon specific and not something that should be done in the core UI/API.  Thinking that we should build the AWS specific `create_bucket_configuration` options in the `raw_cloud_object_store_container_create` method.